app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  belongs_to :post
+end
```

Comment:
Great job @mahdinoori2000 🤝

- [ ] [Optional] As stated in the project requirements, kindly ensure the comments model has a method to update the comments counter for a post.